Microsoft.Packaging.Tools/docs/trimming.md

```diff
@@ -0,0 +1,73 @@
+#Dependency Trimming
+
+This package provides build infrastructure for trimming the output of an application.
+
+It determines what is used by the application by examining static dependencies of the application binary as well as any directly referenced packages.  For any file that is unused it will be removed from the set of files copied to the output and publish folders and removed from the application's dependency file(`deps.json`) in the case of a .NET Core application.
+
+## How to use
+First install the `Microsoft.Packaging.Tools` package in your application.
+
+You must use *Visual Studio 2017* or later, or *.NET Core command-line (CLI) tools 1.0* or later.
+
+### From the commandline
+Specify `/p:TrimUnusedDependencies=true` when building the project with either `dotnet` or `msbuild`.
+
+Examples:
+```
+dotnet build /p:TrimUnusedDependencies=true
```

Comment:
Build produces a deps file.  If you want that deps file trimmed it should be specified.  Not doing so could give you different behavior when building/debugging vs publish.